shared/src/commonMain/kotlin/org/hyperskill/app/step_completion/presentation/StepCompletionFeature.kt

```diff
@@ -9,26 +9,39 @@ interface StepCompletionFeature {
         fun createState(step: Step, stepRoute: StepRoute): State =
```

Comment:
Let's add currentStepRoute to the state. At the moment it is passed as an argument to the reducer for some reason despite it is passed to the createState method.

Reply:
I don't think that it's good idea to have state that we won't change at all. From my point of view it's better to put such kind of data to reducer where it can't be changed via `.copy(...)` method

Reply:
PBR